app/src/main/res/values-v21/styles.xml

```diff
@@ -18,11 +17,18 @@
 
 <resources>
 
-  <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
-    <!-- Customize your theme here. -->
-    <item name="colorPrimary">@color/primary</item>
-    <item name="colorPrimaryDark">@color/primary_dark</item>
-    <item name="colorAccent">@color/accent</item>
-    <item name="android:statusBarColor">?android:attr/colorPrimary</item>
-  </style>
+    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
```

Comment:
The indentation here is off. Please run `git checkout -- .idea` after starting Studio so that you get the shared `.idea/codeStyleSettings.xml`.